From ea0e2066311953ef12703e5ef97706c1a81f315d Mon Sep 17 00:00:00 2001
From: DarkLord1731 <darklord1731@gmail.com>
Date: Sun, 24 Jun 2018 07:56:33 +0000
Subject: [PATCH 26/27] SystemUI: Make notification panel opaque

This helps to prevent lags in the panel on devices which can't handle the transperancy efficiently

Change-Id: I024913ace2861255af160c98bee2e2476762b781
---
 packages/SystemUI/res/color/qs_background_dark.xml | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/packages/SystemUI/res/color/qs_background_dark.xml b/packages/SystemUI/res/color/qs_background_dark.xml
index c19fa080324..5abddf1af0a 100644
--- a/packages/SystemUI/res/color/qs_background_dark.xml
+++ b/packages/SystemUI/res/color/qs_background_dark.xml
@@ -15,6 +15,5 @@
 -->
 
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
-    <item android:alpha="0.87"
-          android:color="?android:attr/colorBackgroundFloating"/>
+    <item android:color="?android:attr/colorBackgroundFloating"/>
 </selector>
-- 
2.17.1

